ios/Podfile.lock

```diff
@@ -199,7 +199,7 @@ PODS:
     - React
   - react-native-geolocation (2.0.2):
     - React
-  - react-native-safe-area-context (3.0.5):
```

Comment:
not sure this change is necessary to commit.

Reply:
ah - good catch, this one snuck in unintentionally